src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/JudicialUserProfileProcessor.java

```diff
@@ -61,25 +61,25 @@ public void process(Exchange exchange) {
         audit(judicialUserProfileJsrValidatorInitializer, exchange);
 
         //Get Elink ids from current load
-        validElinksInUserProfile = filteredJudicialUserProfiles.stream()
-            .map(JudicialUserProfile::getElinksId).collect(toSet());
+        validPerInUserProfile = filteredJudicialUserProfiles.stream()
```

Comment:
same as above rename validPerInUserProfile as validPerIdInUserProfile. Trivial but lets be consistent

Reply:
Hi @abhijitdiwan , this change has been implemented.